client/src/components/EstatePage.js

```diff
@@ -1,12 +1,50 @@
 import "./EstatePage.css";
+import { useState, useEffect } from "react";
+import { Link, useParams } from "react-router-dom";
 import Filter from "../components/Filter";
 import RentBuy from "../components/RentBuy";
 
 export default function EstatePage() {
+  const [properties, setProperties] = useState([]);
+  const { id } = useParams();
+
+  useEffect(() => {
+    const url = `http://localhost:4000/properties/`;
+
+    fetch(url)
+      .then((res) => res.json())
+      .then((apiData) => {
+        setProperties((property) => {
+          return [...property, ...apiData];
+        });
```

Comment:
Nicely done with the state update there! However, this arrow function only really makes sense if `useEffect` had a dependency and the state would need to consider the previous value of the state. `property` seems a strange name then as well since the previous state will always be an array, i.e. a list of things